1-Draft/RFC0000-Experimental-Feature-User-Experience.md

```diff
@@ -0,0 +1,87 @@
+---
+RFC: RFC0000
+Author: Steve Lee
+Status: Draft
+SupercededBy: N/A
+Version: 1.0
+Area: Microsoft.PowerShell.Core
+Comments Due: 12/1/2018
+Plan to implement: Yes
+---
+
+# Experimental Feature User Experience
+
+[Experimental Feature Flags](https://github.com/PowerShell/PowerShell-RFC/blob/master/5-Final/RFC0029-Support-Experimental-Features.md) enable developers to expose experimental features to users to gather feedback before finalizing design.
+That feature was focused on the developer and did not make it easy for users to discover and enable experimental features.
+This RFC addresses the user experience.
+
+## Motivation
+
+    As a PowerShell user,
+    I can discover and enable experimental features,
+    so that try new capabilities safely and provide feedback.
+
+## Specification
+
+### Get-ExperimentalFeature *Breaking Change*
+
+Currently, `Get-ExperimentalFeature` requires the `-ListAvailable` switch to enumerate available experimental features.
+This was modeled after `Get-Module -ListAvailable`.
+Without the switch, `Get-ExperimentalFeature` only shows the enabled experimental features, which by default is none.
+Most users will try this cmdlet without the switch and assume there are no experimental features to try.
+
+Since there are likely not many experimental features available at any point in time and less enabled,
+it would make sense to remove the `-ListAvailable` switch and simply display all experimental features.
```

Comment:
👍

Reply:
Sure, I guess then it behaves like get-windowsfeature. So there's precedent. I'm all for extra visibility here.